components/card/index.tsx

```diff
@@ -15,6 +15,7 @@ export { CardGridProps } from './Grid';
 export { CardMetaProps } from './Meta';
 
 export type CardType = 'inner';
+export type CardSize = 'default' | 'middle' | 'small';
```

Comment:
`middle` sounds strange, maybe better `export type CardSize = 'default' | 'small' | 'large';`

Reply:
I agree, but I used the same sizes as the Table component, and I think it is better to be consistent

Reply:
In list used `default, small, large`: https://ant.design/components/list/ maybe table will be updated in future

Reply:
Hmmmm, I see. The problem is that I can't rename `default` to `large`, because people are already using the default sized cards. So I think `default` has to stay the same. But I can change `middle` to `medium` if you want

Reply:
`large` can't be bigger than `default`?

Reply:
Here is the preview so you can see the current sizes: https://deploy-preview-13180--ant-design.netlify.com/components/card/#components-card-demo-basic

I don't think it would make sense to have a `large` size that is bigger than `default`. `default` already has a lot of padding. Another solution would be to just remove the `middle` size and only have `default` and `small`. But I don't think it's worth doing that just for consistency

Reply:
@ilanus Right, I think default and small would be enough, the small size could go with `8px 16px` as header's padding and `16px` as body's.

Reply:
@ndbroadbent you can make the adjustments to support only default and small 👍